src/components/Uploader/index.js

```diff
@@ -154,7 +154,7 @@ const PassportUploadIntro = ({
       >
         <button
           type="button"
-          className={theme.link}
+          className={classNames(theme['link'], style.buttonLinkUploadCopy)}
```

Comment:
Minor: The change to bracket notation for `theme.link` isn't necessary here. Can be left as it was for consistency.

Reply:
I first tried with `classNames(theme.link…)` but somehow it wasn't working. 😕  But I tried again today and everything worked. 🤷🏼